stdlib/Distributed/src/cluster.jl

```diff
@@ -513,12 +513,16 @@ function set_valid_processes(plist::Array{Int})
     end
 end
 
+"""
+    default_addprocs_params(mgr::ClusterManager) -> Dict{Symbol, Any}
+
+Implemented by cluster managers. The default keyword parameters passed when calling
+`addprocs(mgr)`. The minimal set of options is available by calling
+`default_addprocs_params()`
+"""
+default_addprocs_params(::ClusterManager) = default_addprocs_params()
 default_addprocs_params() = Dict{Symbol,Any}(
     :topology => :all_to_all,
-    :ssh      => "ssh",
-    :shell    => :posix,
-    :cmdline_cookie => false,
```

Comment:
IIRC this is supposed to be a default one and is also used elsewhere, but it has been a long time since I last looked at this.

Reply:
It was only added in #38353, and is only used in `SSHManager`